### ISSUE TITLE
Keyboard Accessibility is Broken

### ISSUE THREAD

**Issue by bocampbell** [carbon-design-system/carbon-website#258]
Unable to tab through from left navigation to tab on the right. Was working in previous versions.

**Comment by alisonjoseph** [comment]
Working on some nav updates will add this to my todo list to look at 👍

**Comment by alisonjoseph** [comment]
will be fixed when we deploy the site next @bocampbell probably on Monday.
